cost-reporter/tests/test_stacked_bar.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+import pytest
+import stacked_bar
+
+FAKE_DATE = datetime.datetime(2020, 12, 25, 17, 5, 55)
+
+
+@pytest.fixture
+def patch_datetime_today(monkeypatch):
```

Comment:
instead of monkey patching datetime, i can recommend you to use the Pyhon lib `freezegun`
https://github.com/spulec/freezegun

Reply:
Awesome, wasn't aware of that lib!

Reply:
Thats awesome and so handy, thanks!